_posts/2021-06-22-solidity-0.8.6-release-announcement.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.6 Release Announcement'
+date: '2021-06-22'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.6](https://github.com/ethereum/solidity/releases/tag/v0.8.6) fixes some
+non-critical but annoying bugs, especially a warning about unreachable code that
+is in fact reachable.
+
+## Bugfixes
+
+### Unreachable Warning
+
+With Solidity 0.8.5, we released a new feature that detects if code is unreachable
+because a called function reverts unconditionally. This detection contained a bug
+where the "reverting behaviour" of a function was not properly updated after a
+function called by it was determined to be not always reverting.
```

Comment:
Maybe we should mention that it affected only code using internal library functions?